src/pybind/mgr/dashboard/tox.ini

```diff
@@ -177,3 +178,13 @@ description = Generate Sphinx documentation from OpenAPI specification
 deps = -r../../../../admin/doc-requirements.txt
 changedir = ../../../../doc
 commands = sphinx-build -W -b html -c . -D suppress_warnings=ref.* -d {envtmpdir}/doctrees mgr/ceph_api {envtmpdir}/html
+
+[testenv:gen-nvmeof-proto]
+allowlist_externals = cp
+changedir = ../
```

Comment:
That was the tricky thing to obtain the proper import path?

Reply:
the import should be like `dashboard.services.proto` inside the proto files or this would give out an error. So had to generate the proto from outside the dashboard folder.